tests/common/include/tests/common/material_interaction.inl

```diff
@@ -254,7 +254,7 @@ TEST(material_interaction, telescope_geometry_energy_loss) {
         matrix_operator().template zero<e_bound_size, e_bound_size>();
 
     // bound track parameter at first physical plane
-    const bound_track_parameters<transform3> bound_param(1u, bound_vector,
+    const bound_track_parameters<transform3> bound_param(0u, bound_vector,
```

Comment:
@niermann999 Is `0u`  the fist "sensitive" surface?

Reply:
Yes, it is